### PR TITLE
Calendar1

### PR DESCRIPTION
**Pull request by domiiii11**

### CODE REVIEW

File: .idea/Java-Homework.iml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
-  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_8">
+  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_14">
```
Comment (Almantask) [inline .idea/Java-Homework.iml:3]: You don't have to commit all. This can be simply not selected in the changelist.

File: .idea/compiler.xml [hunk 2]
```diff
@@ -10,7 +10,7 @@
       </profile>
     </annotationProcessing>
     <bytecodeTargetLevel>
-      <module name="Java-Homework" target="1.8" />
+      <module name="Java-Homework" target="14" />
```
Comment (Almantask) [inline .idea/compiler.xml:13]: You don't have to commit all. This can be simply not selected in the changelist.

File: pom.xml [hunk 3]
```diff
@@ -53,6 +53,14 @@
                     </dependency>
                 </dependencies>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>14</source>
+                    <target>14</target>
+                </configuration>
```
Comment (Almantask) [inline pom.xml:62]: You don't have to commit all. This can be simply not selected in the changelist.

File: src/main/java/Calendar1/Cal1.java [hunk 4]
```diff
@@ -0,0 +1,11 @@
+package Calendar1;
+
+public class Cal1 {
```
Comment (Almantask) [inline src/main/java/Calendar1/Cal1.java:3]: No idea what are you calculating

File: src/main/java/Calendar1/Main.java [hunk 5]
```diff
@@ -0,0 +1,108 @@
+package Calendar1;
+
+public class Main {
+    public static void main(String[] args) {
+        int[] input = new int[]{
```
Comment (Almantask) [inline src/main/java/Calendar1/Main.java:5]: Are all those numbers necessary, what are you calculating?

File: src/main/java/NumberConverter/Converter.java [hunk 6]
```diff
@@ -0,0 +1,43 @@
+package NumberConverter;
+
+import java.util.Arrays;
+
+public class Converter {
```
Comment (Almantask) [inline src/main/java/NumberConverter/Converter.java:5]: This probably shouldn't have been in this PR

File: src/main/java/NumberConverter/Main.java [hunk 7]
```diff
@@ -0,0 +1,11 @@
+package NumberConverter;
```
Comment (Almantask) [inline src/main/java/NumberConverter/Main.java:1]: Not a part of this pr

File: src/main/java/Calendar1/Cal1.java (hunk 4 above)
Comment (Almantask) [reply src/main/java/Calendar1/Cal1.java:3]: I see, it's the advent calculator. You really sohuld have attached the task description with the PR

File: src/main/java/Calendar1/Cal1.java [hunk 8]
```diff
@@ -0,0 +1,11 @@
...
+   public static int calculate(int [] a){
```
Comment (Almantask) [inline src/main/java/Calendar1/Cal1.java:4]: What's a?

File: src/main/java/Calendar1/Cal1.java [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+       int result = 0;
```
Comment (Almantask) [inline src/main/java/Calendar1/Cal1.java:5]: What does the result refer to?

File: src/main/java/Calendar1/Cal1.java [hunk 10]
```diff
@@ -0,0 +1,11 @@
...
+       for(int i = 0; i < a.length; i++){
+          result += ((int) ((Math.ceil(a[i] / 3))) - 2);
```
Comment (Almantask) [inline src/main/java/Calendar1/Cal1.java:7]: What is this formula?

File: src/main/java/Calendar1/Main.java (hunk 5 above)
Comment (Almantask) [inline src/main/java/Calendar1/Main.java:5]: What is the input for?

File: src/main/java/Calendar1/Main.java [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+                88200,
+                86443,
+                79786,
+                131067,
+                105919,
+                126045,
+                135292,
+                117451,
+                67730};
+        System.out.println(Cal1.calculate(input));
```
Comment (Almantask) [inline src/main/java/Calendar1/Main.java:106]: What are you calculating?

File: src/main/java/Demo.java [hunk 12]
```diff
@@ -1,7 +1,64 @@
 package main.java;
 
+import java.util.Arrays;
+import java.util.Scanner;
+
 public class Demo {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        calculator1();
```
Comment (Almantask) [inline src/main/java/Demo.java:8]: Not noun, verb

**Review by Almantask — CHANGES_REQUESTED** [review]

File: .idea/Java-Homework.iml (hunk 1 above)
Comment (domiiii11) [reply .idea/Java-Homework.iml:3]: Sorry, was nervous about not successful pushing for 20 times.:)

File: .idea/compiler.xml (hunk 2 above)
Comment (domiiii11) [reply .idea/compiler.xml:13]: ok! Sorry.

File: pom.xml (hunk 3 above)
Comment (domiiii11) [reply pom.xml:62]: ok :(

File: src/main/java/Calendar1/Cal1.java (hunk 10 above)
Comment (domiiii11) [reply src/main/java/Calendar1/Cal1.java:7]: Why this formula is wrong? Taking numbers from array, dividing from 3, rounding to ceiling, and minus two.

File: src/main/java/Calendar1/Main.java (hunk 5 above)
Comment (domiiii11) [reply src/main/java/Calendar1/Main.java:5]: Array with input number from task. Why it's wrong?

File: src/main/java/Demo.java (hunk 12 above)
Comment (domiiii11) [reply src/main/java/Demo.java:8]: ok.

File: src/main/java/Calendar1/Main.java (hunk 5 above)
Comment (Almantask) [reply src/main/java/Calendar1/Main.java:5]: In the context of a rocket, it has a meaning better than semantic input

File: src/main/java/Calendar1/Cal1.java (hunk 10 above)
Comment (Almantask) [reply src/main/java/Calendar1/Cal1.java:7]: I didn't say it's wrong, just asked what it is

**Pull request closed without merging**